packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -35,10 +37,13 @@ async function updatePackageDependency(
     process.exit(1);
   }
   try {
-    const dependencyType = devDependency ? "devDependencies" : "dependencies";
+    let dependencyType = DEV_DEPENDENCIES;
     const packageJson = JSON.parse(fs.readFileSync(packagePath, "utf-8"));
-    const toVersion = version || (await latestVersion(packageName));
+    if (packageJson.dependencies[packageName]) {
+      dependencyType = DEPENDENCIES;
+    }
     const currentVersion = packageJson[dependencyType][packageName];
```

Comment:
Don't we need to check if it's in both, otherwise this could fail?

Reply:
It defaults to devDependencies. It then checks if it is in dependencies. If it is in dependencies, it will use that. Otherwise will use devDependencies

Reply:
Right, but it checks if the package is in devDeps, if not, it then assumes it's in deps but doesn't actually verify that the package actually exists there.

Reply:
This covers all cases.

For upgrade it will:

1. install in devDependencies if not found
2. upgrade in devDependecies if it is there
3. upgrade in dependencies if it is there

For remove it will:
1. remove if it is in devDependencies
2. remove if it is in dependencies

Reply:
It will also show a unique log 'Installing ${packageName' if the package doesn't exist at all, and 'Upgrading to' if it is found and is upgrading

Reply:
Made some changes

If user has sites-components or react-components in their package.json, we will install pages-components. If they have pages/components imports, sites-components imports, or react-components imports, we also install pages-components. If they already have pages-components, we upgrade it regardless.

All other packages we only upgrade if they already exist in devDependencies or dependencies